### PR TITLE
Google posts insight deprecation

### PR DESCRIPTION
**Pull request by huangalexpartoo**

### CODE REVIEW

File: rest_api/v2/components/index.yml [hunk 1]
```diff
@@ -2356,11 +2356,17 @@ schemas:
 
   GooglePostViewCount:
     description: The number of times the local post was viewed on Google Search
+                 </br>
+                 ⚠️ Due to Google depreciation, This information won't be updated anymore after 20/02/2023.
```
Comment (psykeron) [inline rest_api/v2/components/index.yml:2360]: Due to a Google deprecation.

File: rest_api/v2/components/index.yml [hunk 2]
```diff
@@ -2356,11 +2356,17 @@ schemas:
...
+                 Old insights will still be sent.
     type: integer
     example: 200
 
   GooglePostClickCount:
     description: The number of times the local post was viewed on Google Search
+                 </br>
+                 ⚠️ Due to Google depreciation, This information won't be updated anymore after 20/02/2023.
```
Comment (psykeron) [inline rest_api/v2/components/index.yml:2368]: Due to a google deprecation,

File: rest_api/v2/components/index.yml [hunk 3]
```diff
@@ -2590,13 +2596,16 @@ schemas:
...
+    description: Publisher related to the post
     enum:
       - 'google_my_business__posts'
       - 'facebook__posts'
-    example: google_my_business__posts
+    example: facebook__posts
   postInsight:
     description: Statistics related to the posts on the different platforms.
+                 </br>
+                 ⚠️ Due to Google depreciation, on 20/02/2023, Google views and counts won't be available anymore.
```
Comment (psykeron) [inline rest_api/v2/components/index.yml:2607]: Due to a Google deprecation,

File: rest_api/v2/components/index.yml (hunk 1 above)
Comment (huangalexpartoo) [reply rest_api/v2/components/index.yml]: oups

File: rest_api/v2/components/index.yml [hunk 4]
```diff
@@ -2357,15 +2357,15 @@ schemas:
   GooglePostViewCount:
     description: The number of times the local post was viewed on Google Search
                  </br>
-                 ⚠️ Due to Google depreciation, This information won't be updated anymore after 20/02/2023.
+                 ⚠️ Due to Google deprecation, This information won't be updated anymore after 20/02/2023.
```
Comment (psykeron) [inline rest_api/v2/components/index.yml:2360]: also, don't forget the `a` before `Google deprecation`,  and the lower case `t` in `this information`;)

File: rest_api/v2/components/index.yml [hunk 5]
```diff
@@ -2357,15 +2357,15 @@ schemas:
...
                  Old insights will still be sent.
     type: integer
     example: 200
 
   GooglePostClickCount:
     description: The number of times the local post was viewed on Google Search
                  </br>
-                 ⚠️ Due to Google depreciation, This information won't be updated anymore after 20/02/2023.
+                 ⚠️ Due to Google deprecation, This information won't be updated anymore after 20/02/2023.
```
Comment (psykeron) [inline rest_api/v2/components/index.yml:2368]: same

File: rest_api/v2/components/index.yml [hunk 6]
```diff
@@ -2604,7 +2604,7 @@ schemas:
   postInsight:
     description: Statistics related to the posts on the different platforms.
                  </br>
-                 ⚠️ Due to Google depreciation, on 20/02/2023, Google views and counts won't be available anymore.
+                 ⚠️ Due to Google deprecation, on 20/02/2023, Google views and counts won't be available anymore.
```
Comment (psykeron) [inline rest_api/v2/components/index.yml:2607]: just the a

Comment (huangalexpartoo) [reply rest_api/v2/components/index.yml:2607]: not sure to understand

**Review by psykeron — APPROVED** [review]
💯

**Pull request merged**